KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/AuthenticationModule.java

```diff
@@ -15,16 +15,23 @@
 import com.wowza.wms.stream.IMediaStream;
 import com.wowza.wms.stream.MediaStreamActionNotifyBase;
 import com.wowza.wms.stream.live.MediaStreamLive;
+import com.wowza.wms.vhost.IVHost;
+import com.wowza.wms.vhost.VHost;
 import org.apache.log4j.Logger;
+import com.wowza.wms.rtp.model.RTPSession;
 import com.kaltura.media_server.services.*;
 
 import java.util.HashMap;
 import java.util.regex.Matcher;
+import com.kaltura.media_server.services.KalturaStreamType;
 
 public class AuthenticationModule extends ModuleBase  {
-
+    private enum StreamType {
```

Comment:
don't see why we need to keep this, you can add a function that determin that from the IMediaStream,
please remove